src/components/views/rooms/ReadReceiptGroup.tsx

```diff
@@ -114,7 +120,7 @@ export function ReadReceiptGroup(
     });
 
     let remText: JSX.Element;
-    const remainder = readReceipts.length - MAX_READ_AVATARS;
+    const remainder = readReceipts.length - maxAvatars;
```

Comment:
think this would be clearer as `remainder = readReceipts.length - avatars.length;` even though functionally identical for results passing `if (remainder > 0)`

Reply:
That wouldn’t really be possible – `avatars` has to contain all avatars, even the hidden ones, to ensure the hidden ones are in the DOM, just hidden(so that if we animate them, they’ll move correctly)

Reply:
Valid point!